numpy/lib/npyio.py

```diff
@@ -1271,8 +1271,8 @@ def genfromtxt(fname, dtype=float, comments='#', delimiter=None,
 
     Parameters
     ----------
-    fname : file or str
-        File, filename, or generator to read.  If the filename extension is
+    fname : file, str, list of str 
+        File, filename, list,  or generator to read.  If the filename extension is
         `.gz` or `.bz2`, the file is first decompressed. Note that
```

Comment:
Might add something like `The strings in a list or produced by a generator are treated as lines`.